app/javascript/actions/bot.js

```diff
@@ -6,40 +6,71 @@ import { pushNotification } from './notifications'
 
 export const UPDATE = 'BOT_UPDATE'
 export const PUBLISH = 'BOT_PUBLISH'
+export const PUBLISH_SUCCESS = 'BOT_PUBLISH_SUCCESS'
 export const UNPUBLISH = 'BOT_UNPUBLISH'
+export const UNPUBLISH_SUCCESS = 'BOT_UNPUBLISH_SUCCESS'
 export const DELETE = 'BOT_DELETE'
 
+export const botUpdate = (bot : T.Bot) : T.BotAction => ({
```

Comment:
I find a bit confusing to have botUpdate and updateBot. But I can't come with a better alterantive 🤷‍♂️ .

maybe not export them and/or add a `_` prefix?

Reply:
I renamed all direct action creators after the constants of their types. The other actions were there before. One of the reasons for explicitly implementing the action creators (as opposed to just `dispatch({type: ...})` is for testing, so not exporting them defeats the purpose (documentation being the other main one IMHO). I'll settle for the `_` prefix :)